packages/plugin-core/src/constants.ts

```diff
@@ -246,6 +246,15 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     // no prefix, we don't want to show this command
     title: `${CMD_PREFIX} Contribute`,
   },
+  GOTO: {
+    key: "dendron.goto",
+    // no prefix, we don't want to show this command
```

Comment:
Just a nitpick, but it looks like this comment has been accidentally getting copy-pasted around a bunch, these commands all do have a prefix and we do want to show them